src/java.base/share/classes/jdk/internal/access/JavaLangInvokeAccess.java

```diff
@@ -113,8 +113,14 @@
      * Used by {@code jdk.internal.foreign.LayoutPath} and
      * {@code jdk.incubator.foreign.MemoryHandles}.
      */
-    VarHandle memoryAddressViewVarHandle(Class<?> carrier, long alignmentMask,
-                                         ByteOrder order, long offset, long[] strides);
+    VarHandle memoryAccessVarHandle(Class<?> carrier, long alignmentMask,
+                                    ByteOrder order, long offset, long[] strides);
+
+    /**
+     * Is {@code handle} a memory access varhandle?
+     * Used by {@code jdk.incubator.foreign.MemoryHandles}.
+     */
```

Comment:
Does this need `@param` and `@return` tags?

Reply:
I went with the style of the other comments surrounding this. This is internal API, seems like the main need (as arised during RFR) is to indicate where a routine is used :-)